src/explorer/views/Farms.vue

```diff
@@ -104,13 +87,7 @@
 <script lang="ts">
 import { Component, Vue, Watch } from "vue-property-decorator";
 import DetailsV2 from "../components/DetailsV2.vue";
-import {
-  filterQuery,
-  getFarmsQuery,
-  IFarm,
-  IFetchPaginatedData,
-  IFilterQuery,
-} from "../graphql/api";
+import { filterQuery, getFarmsQuery, IFarm, IFetchPaginatedData, IFilterQuery } from "../graphql/api";
```

Comment:
`filterQuery`, `IFilterQuery` and `apollo` are no longer used and should be removed